workspaces/tests/account.rs

```diff
@@ -33,3 +33,24 @@ async fn test_subaccount_creation() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[test(tokio::test)]
+async fn test_transfer_near() -> anyhow::Result<()> {
+    let worker = workspaces::sandbox().await?;
+    let (alice, bob) = (
+        worker.dev_create_account().await?,
+        worker.dev_create_account().await?,
+    );
+
+    // transfer 500_000_000 token from alice to bob
```

Comment:
Please, add asserts on that alice's balance has changed (one assert before, and one after the transfer)